src/components/routes/AccountSettings.tsx

```diff
@@ -28,19 +29,19 @@ import API_HOST from '../util/APIHost';
 import { client } from '../util/makeClient';
 import asAPIError from '../util/asAPIError';
 
-import ErrorOutlineIcon from '@material-ui/icons/ErrorOutline';
 import InputAdornment from '@material-ui/core/InputAdornment';
 
 import InstagramIcon from '@material-ui/icons/Instagram';
 import FacebookIcon from '@material-ui/icons/Facebook';
 import TwitterIcon from '@material-ui/icons/Twitter';
 import LinkedInIcon from '@material-ui/icons/LinkedIn';
 import { ForgotPasswordModal } from './ForgotPasswordModal';
-import { Link as RouterLink, Prompt } from 'react-router-dom';
+import { Link as RouterLink, Prompt, useLocation } from 'react-router-dom';
 import Link from '@material-ui/core/Link';
-import { Autocomplete } from '@material-ui/lab';
+import { Alert, Autocomplete } from '@material-ui/lab';
 
 import { matchSorter } from 'match-sorter';
+import qs from 'qs';
```

Comment:
Could you add this as an explicit dependency in package.json? While it exists in the lockfile, since it's being used here it should be defined in package.json itself.

Reply:
Aah, can do. I didn't check if it was in the package.json because I just assumed it was if it was in the lock :P